tests/NLog.UnitTests/ApiTests.cs

```diff
@@ -183,7 +183,7 @@ public void IStringValueRenderer_AppDomainFixedOutput_Attribute_NotRequired()
                 if (typeof(NLog.Internal.IStringValueRenderer).IsAssignableFrom(type) && !type.IsInterface)
                 {
                     var appDomainFixedOutputAttribute = type.GetCustomAttribute<AppDomainFixedOutputAttribute>();
-                    Assert.True(ReferenceEquals(appDomainFixedOutputAttribute, null), $"{type.ToString()} should not implement IStringValueRenderer");
+                    Assert.True(ReferenceEquals(appDomainFixedOutputAttribute, null), $"{type.ToString()} cannot implement IStringValueRenderer as it isn't marked as [AppDomainFixedOutput]");
```

Comment:
Should not implement `IStringValueRenderer` because it is marked as `[AppDomainFixedOutput]`